e2e/tests/workspace/workspace-create.spec.ts

```diff
@@ -7,7 +7,7 @@ require('../../driver/puppeteerExtension');
 import { getCursorValue } from '../../driver/elementHandle-util'
 
 jest.setTimeout(300000);
-describe('Workspace create:', () => {
+describe.skip('Workspace create:', () => {
```

Comment:
Should this be reverted before this PR goes in?

Reply:
reverted. test is broken due to Neha's PR on new workspace UI. It needs to be fixed.